MediaBrowser.Providers/TV/SeriesMetadataService.cs

```diff
@@ -58,7 +58,7 @@ protected override async Task AfterMetadataRefresh(Series item, MetadataRefreshO
             }
             catch (Exception ex)
             {
-                Logger.LogError(ex, "Error in DummySeasonProvider");
+                Logger.LogError(ex, "Error in DummySeasonProvider for {0}", item.Path);
```

Comment:
```suggestion
                Logger.LogError(ex, "Error in DummySeasonProvider for {ItemPath}", item.Path);
```

Reply:
Sounds good.  I've made the changes, rebased, and force pushed the updates.